subsys/logging/backends/log_backend_net.c

```diff
@@ -338,9 +338,18 @@ static void panic(struct log_backend const *const backend)
 	panic_mode = true;
```

Comment:
Typo in commit `logging: backend: net: avoid automatic enabling`
s/fixis/fixes/
s/than/then/
Also please start sentences with capital letters

Reply:
done